src/components/CustomTextArea.tsx

```diff
@@ -1,7 +1,7 @@
-import React, { ReactNode, useCallback, useEffect, useRef } from 'react';
+import React, { ReactNode, useCallback, useEffect, useRef, useState } from 'react';
 
-import { StandardEditorProps, StringFieldConfigSettings } from '@grafana/data';
-import { TextArea } from '@grafana/ui';
+import { StringFieldConfigSettings } from '@grafana/data/field/overrides/processors';
+import { StandardEditorProps } from '@grafana/data';
```

Comment:
I think this more-specific import will make this more breakable in the future. I think revert this one.

Reply:
Reverted.